packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -5,6 +5,38 @@
 @use "../utils/grid-items.scss" as *;
 
 .dataviews-view-grid {
+	&.dataviews-view-grid__content {
+		padding: 0 $grid-unit-60 $grid-unit-30;
+		display: flex;
+		flex-direction: column;
+		gap: $grid-unit-40;
+		container-type: inline-size;
+
+		@media not (prefers-reduced-motion) {
+			transition: padding ease-out 0.1s;
+		}
+
+		@container (max-width: 430px) {
+			padding-left: $grid-unit-30;
+			padding-right: $grid-unit-30;
+		}
+
+		.dataviews-view-grid__row {
+			display: grid;
+			gap: $grid-unit-40;
+
+			.dataviews-view-grid__row__gridcell {
+				outline: 0;
+				outline-style: solid;
```

Comment:
```suggestion
				outline-style: solid;
				border-radius: $radius-medium;
```

It might be godd to add a radius:

| Before | After |
|--------|--------|
| <img width="304" height="256" alt="Image" src="https://github.com/user-attachments/assets/9e8409c7-fef4-42f7-97b5-c5c61fd40f15" /> | <img width="314" height="255" alt="Image" src="https://github.com/user-attachments/assets/9a05e59e-5259-48ba-8e0f-9618cc3e3469" /> |